package.json

```diff
@@ -27,10 +27,10 @@
   "devDependencies": {
     "@babel/core": "^7.10.5",
     "@babel/preset-react": "^7.10.4",
-    "@ckeditor/ckeditor5-dev-bump-year": "^38.0.0",
-    "@ckeditor/ckeditor5-dev-ci": "^38.0.0",
-    "@ckeditor/ckeditor5-dev-release-tools": "^38.0.0",
-    "@ckeditor/ckeditor5-dev-utils": "^38.0.0",
+    "@ckeditor/ckeditor5-dev-bump-year": "^40.0.0",
+    "@ckeditor/ckeditor5-dev-ci": "^40.0.0",
+    "@ckeditor/ckeditor5-dev-release-tools": "^40.0.0",
+    "@ckeditor/ckeditor5-dev-utils": "^40.0.0",
```

Comment:
These changes require regenerating the lock file.

Reply:
Resolved by 13fb6890c0b7a87c42db665efc4233a310fe822c and 597fc0c66868b78e6144deeec02c6d5504d5b0de.